### ISSUE TITLE
plman.ExecutePlaylistDefaultAction bug?

### ISSUE THREAD

**Issue by elia-is-me** [TheQwertiest/foo_spider_monkey_panel#123]
i try to play an item using plman.ExecutePlaylistDefaultAction like:
```
let id = 0; // or some valid playlist index else;
plman.ExecutePlaylistDefaultAction(id, 0 /* or some valid track id */);
```
but it only works when `id` is equal to `plman.ActivePlaylist`.

**Comment by TheQwertiest** [comment]
Thanks for the bug report! I'll take a look.

**Comment by marc2k3** [comment]
This is a bug with fb2k itself and needs reporting. You could hack around it by switching the active playlist manually first but you shouldn't have to.

**Comment by TheQwertiest** [comment]
Yep, marc2k3 is right, it's an easily reproducible fb2k bug, I'll report it on HA.

**Comment by TheQwertiest** [comment]
Reported: https://hydrogenaud.io/index.php?topic=120455
I'll close this issue, since there's nothing I can do.

**Comment by TheQwertiest** [comment]
fixed in 1.6.3: http://www.foobar2000.org/changelog

**Comment by elia-is-me** [comment]
got it.
